Update security.rst

ThomasLandauer opened the pull request:
Minor

ThomasLandauer committed:
Update security.rst

Minor

ThomasLandauer committed:
Update security.rst

Fixing https://github.com/symfony/symfony-docs/runs/524156891

ThomasLandauer commented:
I upgraded heading "3b" one level higher for consistency. Now I'm getting this error:

> /home/runner/work/symfony-docs/symfony-docs/security.rst:370: SEVERE: Title level inconsistent:

This is concerning chapter 4 which I didn't modify.

jpjoao commented on security.rst:
```diff
@@ -342,14 +342,12 @@ Guard Authenticators
 ....................
```
```suggestion
~~~~~~~~~~~~~~~~~~~~
```
Because you changed the title level of item 3b the level of Guard Authenticators needs to be changed too.

ThomasLandauer replied:
Thanks - looks like I finally got it now :-)

ThomasLandauer committed:
Update security.rst

Fixing https://github.com/symfony/symfony-docs/pull/13388/checks?check_run_id=524167834

wouterj reviewed (approved):
Thanks!

javiereguiluz commented:
Thanks Thomas.
